modules/op-using-ssh-authentication-in-git-type-tasks.adoc

```diff
@@ -1,11 +1,30 @@
 // This module is included in the following assemblies:
-// * secure/authenticating-pipelines-using-git-secret.adoc
+// * secure/authenticating-pipelines-repos-using-secrets.adoc
 
 [id="op-using-ssh-authentication-in-git-type-tasks_{context}"]
-= Using SSH authentication in git type tasks
+= Using Git SSH authentication in tasks
 
-When invoking Git commands, you can use SSH authentication directly in the steps of a task. SSH authentication ignores the `$HOME` variable and only uses the user's home directory specified in the `/etc/passwd` file. So each step in a task must symlink the `/tekton/home/.ssh` directory to the home directory of the associated user.
+You can invoke directly Git commands in the steps of a task and use SSH authentication, but an additional step is required.
```

Comment:
Tiny adverb position and [active voice](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice) nit:
```suggestion
You can directly invoke Git commands in the steps of a task and use SSH authentication, but you must perform an additional step.
```